extensions/wikia/FacebookClient/FacebookClientHooks.class.php

```diff
@@ -154,17 +154,13 @@ public static function onSkinTemplatePageBeforeUserMsg( &$html ) {
 	}
 
 	/**
+	 * SUS-905: Load FBTags JS in a single HTTP request with the other groups
+	 * @param OutputPage $out
 	 * @param Skin $skin
-	 * @param string $text
-	 *
-	 * @return bool
-	 * @throws WikiaException
+	 * @return bool true to continue hook processing
 	 */
-	public static function onSkinAfterBottomScripts( $skin, &$text ) {
-
-		$script = AssetsManager::getInstance()->getURL( 'facebook_client_fbtags_js' );
-		$text .= Html::linkedScript( $script[0] );
-
+	public static function onBeforePageDisplay( &$out, &$skin ) {
+		Wikia::addAssetsToOutput( 'facebook_client_fbtags_js' );
```

Comment:
`Wikia::addAssetsToOutput` is skin-agnostic - it just generates an AM URL and injects a script tag into `OutputPage`.